Week2/homework/maartjes_work.js

```diff
@@ -44,4 +44,40 @@ const tuesday = [
 
 const tasks = monday.concat(tuesday);
 
-// Add your code here
+//Map the tasks to durations in hours.
+let Maartje = tasks.map(element => element.duration / 60);
```

Comment:
You should have rounded off the hours over here in order to make your task easier.